apiserver/facades/agent/upgrader/upgrader_test.go

```diff
@@ -157,15 +157,6 @@ func (s *upgraderSuite) TestWatchAPIVersionControllerAgent(c *gc.C) {
 	wc.AssertClosed()
 }
 
-func (s *upgraderSuite) TestUpgraderAPIRefusesNonMachineOrApplicationAgent(c *gc.C) {
```

Comment:
Is there another existing test we can add a unit tag to?
To check for embedded caas units?